FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.FunctionApp/appsettings.json

```diff
@@ -31,5 +31,9 @@
   },
   "scheduler": {
     "maxConcurrencyCount": 10
+  },
+  "healthCheck": {
+    "healthCheckTimeoutInSeconds": 25,
+    "healthCheckTimeIntervalInSeconds": 30
```

Comment:
how do we choose the default values here? Since this is a value from user input, do we need to check the values?

Reply:
Set internal to 60 and timeout 30.